pre-commit

```diff
@@ -25,9 +25,33 @@ if [[ "$1" == "install" ]]; then
     exit
 fi
 
+save_merge_files() {
+    # MERGE_[HEAD|MODE|MSG]
+    for f in "$root"/.git/MERGE_*; do
+	if [[ -e "$f" ]]; then
+            t=$(basename $f)
```

Comment:
This is moving the files into the root.  Is the goal to ensure that `git stash` saves these files?  Add a comment to that effect perhaps.